hudi-utilities/pom.xml

```diff
@@ -241,6 +245,17 @@
       </exclusions>
     </dependency>
 
+    <dependency>
+      <groupId>org.apache.spark</groupId>
+      <artifactId>spark-hive_${scala.binary.version}</artifactId>
+      <exclusions>
+        <exclusion>
+          <groupId>*</groupId>
+          <artifactId>*</artifactId>
+        </exclusion>
+      </exclusions>
+    </dependency>
```

Comment:
No point adding this since all artifacts are excluded?

Reply:
I can try removing again and testing without it but i think for some reason this helped resolve some test failures in this module.

Reply:
Removing this dependency seems to cause failures 
For now opting to keep this dependency.

[ERROR] testBuildHiveSyncConfig{boolean}[1]  Time elapsed: 0.017 s  <<< ERROR!
java.lang.NoClassDefFoundError: org/apache/spark/sql/hive/HiveExternalCatalog
	at org.apache.hudi.DataSourceUtils.buildHiveSyncConfig(DataSourceUtils.java:322)
	at org.apache.hudi.TestDataSourceUtils.testBuildHiveSyncConfig(TestDataSourceUtils.java:261)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)